$OpenBSD$

Make it possible for us to send makexpi.sh a --fast option
from Makefile
--- Makefile.orig	Thu Apr 23 12:23:47 2015
+++ Makefile	Thu Apr 23 12:24:02 2015
@@ -5,7 +5,7 @@
 	./makexpi.sh $@
 # this makes prerelease xpis (and is the default rule)
 prerelease: pkg
-	./makexpi.sh
+	./makexpi.sh $(MAKEXPI_ARGS)
 pkg:
 	mkdir pkg
 clean:
